src/autolabel/models/google.py

```diff
@@ -73,6 +74,7 @@ def __init__(
         model_params = config.model_params()
         self.model_params = {**self.DEFAULT_PARAMS, **model_params}
         self.llm = VertexAI(model_name=self.model_name, **self.model_params)
+        self.vertexaiModel = generative_models.GenerativeModel(self.model_name)
```

Comment:
hope this doesnt make some connection to the llm api as well

Reply:
I do not know if it makes a connection but it does make a post request but according to @nihit it was fine. Maybe I am missing something?

Reply:
sg